BootloaderCommonPkg/Include/Library/ShellLib.h

```diff
@@ -11,7 +11,7 @@
 #include <PiPei.h>
 #include <Library/BaseLib.h>
 
-#define MAX_COMMAND_LINE_LEN 256
+#define MAX_COMMAND_LINE_LEN  80
```

Comment:
80 may be not long enough for full shell. e.g. to append new command line parameters.
use 80 only when mini shell PCD is defined?

Reply:
Make sense.